diff --git a/node_modules/@trezor/connect-webextension/build/trezor-connect-webextension.js b/node_modules/@trezor/connect-webextension/build/trezor-connect-webextension.js
index b635710..134fd8f 100644
--- a/node_modules/@trezor/connect-webextension/build/trezor-connect-webextension.js
+++ b/node_modules/@trezor/connect-webextension/build/trezor-connect-webextension.js
@@ -2495,7 +2495,7 @@ class PopupManager extends (events_default()) {
         // create new window above instead of opening new tab
         if (currentWindow.type !== 'normal') {
           chrome.windows.create({
-            url
+            url, type: 'popup'
           }, newWindow => {
             chrome.tabs.query({
               windowId: newWindow?.id,
