diff --git a/node_modules/@trezor/connect-web/lib/popup/index.js b/node_modules/@trezor/connect-web/lib/popup/index.js
index 46e3465..826ae9d 100644
--- a/node_modules/@trezor/connect-web/lib/popup/index.js
+++ b/node_modules/@trezor/connect-web/lib/popup/index.js
@@ -175,7 +175,7 @@ class PopupManager extends events_1.default {
             chrome.windows.getCurrent(currentWindow => {
                 this.logger.debug('opening popup. currentWindow: ', currentWindow);
                 if (currentWindow.type !== 'normal') {
-                    chrome.windows.create({ url }, newWindow => {
+                    chrome.windows.create({ url, type: 'popup' }, newWindow => {
                         chrome.tabs.query({
                             windowId: newWindow === null || newWindow === void 0 ? void 0 : newWindow.id,
                             active: true,
